package.json

```
@@ -1,7 +1,8 @@
 {
-    "name": "ESLint",
+    "name": "eslint-website",
```

Maybe better to match the repo name. 

```suggestion
    "name": "eslint.org",
```